### DISCUSSION TITLE
Artist-producer looking for interactive developer for inclusive, interactive website version of a live multi-disciplinary show

### DISCUSSION THREAD

**Discussion by leslieting** [pair-cg/general-contributions#101] [User Stories]
# 1. Motivating Story
_What's the story here? Please describe why you have a particular need._

I have a multi-disciplinary production (a music-drive theatre performance) that is in development and for a live, in-person audience called, What Brings You In (WBYI). 

I would also like to create an interactive, inclusive website, corresponding to the live show. We'll be finding ways to translate the different intentions of the live show to an interactive website that considers audiences with a broader range of sensory experiences (vision, hearing and neuro-diversity).

(Side note on other inclusivity: Bandwidth access? ie creating light websites; multi-lingual)

Key is feeling connected to other users/audience members (ie. not just a lone user on a website)

Scope of the prototype project can be negotiated - ie. we create a prototype for one or two parts versus all six parts and go more in-depth with the inclusive interactivity

**************

Description of the live, in-person version of WBYI. The desired live audience experience is described followed by a preliminary user experience translation.

The live show has 3 main elements:

- music and sound design (this drives the concept and intention)
- theatrical lighting design and live video (another character)
- interviews (another sound element, contextualizes the music and its intentions between parts)

The way the elements are used for the intended concept are outlined below (this work is still in development). Interview clips of lay people, therapists are interweaved between the parts of the show, to be sound designed in relation to the sound and music of each part for arc of the overall show and context. 

BEGINNING OF SHOW
An interview clip will be heard to set the context of habit/pattern/repetition.

**Part I.** Two people: Movements 4 and 5 of Dirt Road for violin & percussion composed by Linda Catlin Smith
Performers: Violinist - Leslie Ting; Percussionist - Germaine Liu; Live video performer - Elysha Poirier

Performers are in back, right of the stage. Instruments are very lightly amplified if needed depending on the venue's acoustics.
Live video art is projected on a screen at the back of the stage of a hand drawing a path in dirt. View of this dirt road video starts as a narrow horizontal band across middle of the screen and slowly lengthens out to fill the screen over the course of the music of Part I.
  
Intent: Dirt Road bookends the whole performance, repeated at the end. This piece is used as an analogy of two figures on a journey together - ex. self and inner child, a person and therapist, self and parent. As the beginning fo the show, it is the entry point.

Intended live audience experience: Feeling of entry, beginning into the show. Sense of repetition.
Possible user experience translation: Feeling of entry, beginning, sense of repetition in your daily life. 

TRANSITION: Interview clip about how we connect with another person. What is the most yourself you can be with another person?

**Part II.** Two people connecting: Improvised music by violin & percussion 
Performers: Violinist - Leslie Ting; Percussionist - Germaine Liu

Performers are seated facing each other in the centre front of the stage. 
No video, theatrical lighting is simple and focused on the performers. 

Intent: The audience witnesses two people connecting wordlessly through their instruments through the practice of improvised music, like a conversation between two people. The elements (feeling of safety, listening, presence or engagement) of a "successful" an improvised duo are similar to the success of a spoken conversation.

Intended live audience experience: Listening to a duo improvisation in the context of an intimate conversation between two people
Possible user experience translation: Being in intimate conversation (responding, revealing, listening) with another person

TRANSITION: Interview clip about what's in the subconscious, below the surface.

**Part III.** Sandplay: sound and light abstraction of sandplay therapy
Performers: Percussionist - Germaine Liu; Live video artist - Elysha Poirier 

Germaine is at a sand-light box in the centre of the stage. Elysha is stage right of Germaine and the sandbox, live manipulating the image from the infrared camera pointing down on Germaine's hands and the sandbox. A theatre light points up through the box from underneath. Contact microphones are connected to the box at all four corners, mapped to the surround sound speakers around the room. 

Intent: This piece is about surfacing things from the subconscious. Germaine performs a sound piece in the sandbox, using it like a percussion instrument, while Elysha live manipulates the large projected image of Germaine's hands in the sand box. The audience is invited into this liminal space visually and through the sound of the sand coming from the surrounding speakers.

Intended live audience experience: Listening, and being included in the process of revealing something about a person (ie. the performer)
Possible user experience translation: Feeling of making something, which may or may not reveal something. Are you creating or revealing? 

TRANSITION: Interview clip about grappling with yourself, trying to understand your own inner workings. 

**Part IV.**  Person & Self: Commissioned piece, "Beholding" for violin & electronics composed by Rose Bolton 
Performer: Violinist - Leslie Ting

Leslie performs this piece facing a speaker, stage left, which emits the composed electronics, in profile to the audience.
No video. The lighting focused on stage left, between the performer and the speaker, which is like a character.

Intent:  The speaker and electronics represent the mind the person/violin is interacting/struggling with. The sound is localized and perceived from stage left. The audience witnesses someone grappling with themselves. 

Intended live audience experience: Watching and listening to someone grapple with themselves - the mystery, the uncertainty.
Possible user experience: Feeling of mystery, uncertainty. Self-reflection.

TRANSITION:  The electronic sounds from the "self-speaker" on stage in Part IV moves seamlessly into speakers placed in and around the audience. Leslie moves to centre front of the stage and faces the audience while this is happening. 

**Part V.** Us: Commissioned piece, "Part V for What Brings You In" for violin & electronics by Julia Mermelstein 
Performer: Violinist - Leslie Ting

No video. The theatrical lighting opens up more and includes the audience. 

Intent: This represents a shift towards an awareness of the collective. Being an individual in systems that are made up of other individuals.

Intended live audience experience: The transition from Part IV to Part V is about connecting with other audience members and the performers equally.
Possible user experience : Feeling connected with other users and the performers equally.

**Part VI.** Two people: Movements 13 and 11 of Dirt Road
Performers: Violinist - Leslie Ting; Percussionist - Germaine Liu; Live video performer - Elysha Poirier

Performers return to the back, right of the stage. Instruments are very lightly amplified. 
Live video art of a close up of a hand drawing a path in a dirt is projected on a large screen back of stage. View of dirt road video starts full screen and re-narrows or morphs.
  
Intent: Return of the lesson. By repeating Dirt Road at the end but with different movements, it also references the spiral nature of growing as a person - a lesson returns but things are not the same. Over the course of this performance, time has passed, and we've explored our selves. Have we grown? Have we learned from the process? 

Intended live audience experience: Feeling of return, familiarity, continuing on.
Possible user experience: Return, familiarity, continuing on. 

# 2. Need:
_Now the why is out of the way, what is it you actually need?_

To create a job posting for an interactive developer.
To find job sites that will have the relevant people in their networks.

# 3. Action:
_What is an action that can be taken towards fulfilling that need?_

1.  Describe the live show and the desired audience and user feeling we are aiming for in the website prototype
2.  Describe additional useful domain knowledge
3.  Define duration of engagement
4.  Define the fee offered
5.  List the deliverables/output
6.  Find sites for finding a relevant pool of candidates

# 4. Desired Outcome:
_What is the desired outcome?_

To define the scope of the prototype digital project for a job posting

# 5. Perspective:
_From what perspective is this user story coming from?_

Able-bodied creator-producer

# 6. Stakeholder Group:
_Which stakeholder groups are impacted by this User Story?_

Creator-producer
UX designers
Job search site
Potential interactive developers

# 7. Additional info:
_Here's more information._

https://www.leslieting.com/what-brings-you-in

What Brings You In (WBYI) is the latest multi-disciplinary work by Leslie Ting Productions. Since 2007, Leslie has periodically gone to therapy, always finding the process strange, but personally necessary. WBYI was born out of a curiosity about what compelled others to go to therapy. Her research led to WBYI’s broader theme of self-understanding in relation to others. Initial development included extensive hands-on research and 40 field interviews with psychologists, counselors, alternative therapists and lay people. A one-hour, music-driven performance piece created by Leslie Ting Productions (LTP) and performed/directed by Leslie (violin), Germaine Liu (percussion) and Elyshia Poirier (live video), WBYI explores the concept of “what brings us in” - to a concert hall, a communal space, our deeper nature. 

The in-person experience of WBYI spans artistic disciplines of live video art, immersive sound and lighting design to express the paradox of discovering oneself through the experience of connecting with others. Comprised of six vignettes, each centers around a musical performance with integrated staging, lighting and sound spatialization, all animating the underlying WBYI thematic material of self-exploration. Clips of the one-on-one field research interviews are weaved throughout the vignettes and contextualize the overall arc of the piece. 

# 8. Competency Questions:

CQ: What is the job title?
Answer: interactive developer
Proposed terminology: job title
Instance of terminology: interactive developer

CQ: What is the length of the professional engagement?
Answer: 1 year
Proposed terminology: contract duration
Instance of terminology: 1 year

CQ: What is the fee for this engagement?
Answer: $15 000
Proposed terminology: fee
Instance of terminology: $15 000

CQ: What additional domain knowledge is considered an asset? 
Answer: music, theatre, performing arts
Proposed terminology: additional domain knowledge
Instance of terminology: music, theatre, performing arts

CQ: What is the language of communication?
Answer: English
Proposed terminology: language
Instance of terminology: English

CQ: What is the deliverable or output of this engagement?
Answer: a prototype of an interactive, inclusive website of some portion of the entire show
Proposed terminology: deliverable
Instance of terminology: prototype

**Comment by marPAIR** [comment]
The detailed breakdown of the show may actually be helpful in developing user story #100. Are there pieces of information that we can extract or translate into "relevant information" for the differently-abled groups?

1. The need could be more specific, perhaps creating a job description or a request for collaboration? And what is needed, is to define the hard and soft skills that are required for this collaboration, without predetermining the technical requirements? In addition to the information, do you also need to know where to actually distribute the request so it is seen by the right kinds of people?
2. The desired outcome could even be taken one step back, as this user story covers a lot of ground in terms of process. Is there a more specific outcome that we could focus on, for example just defining the scope of the digital project itself? That needs to be done in order to inform basic constraints like timeline, budgets, and of course actually reaching and being accessing to the target audiences.

In terms of competency questions, here are a few ideas:

1. What is the length of the professional engagement?
2. What is the maximum budget for this engagement?
3. What domain knowledge is considered an asset?
4. What is the deliverable or output of this engagement?

**Reply by leslieting** [reply]
Yes, I am going for a job description, which would be easier if I knew what exactly we wanted to do, but that work hasn't actually been done yet! Even the live show that this would be based on is only maybe 65% developed. It's been started though, I have two UX designers already and we have done some preliminary discussions on how the intent of the live show could be translated to an interactive experience. 

I also don't know much about this work and the skills I'd want, I did have a convo last week with one of my UX designers to help me understand better the language and the working relationship between us and a developer. 

Yes, I'd love to know better, more targeted sites for this kind of work than LinkedIn!

**Reply by leslieting** [reply]
@marPAIR Should I remove the long description from the "motivating story" section?
